max_depth: 3000
publish_tf: true
kernel_size: -1
use_align: true